Cargo.toml

```diff
@@ -23,12 +23,16 @@ users = "0.11.0"
 page_size = "0.4.2"
 serde = {version = "1.0.102", features = ["std", "derive"], optional = true}
 zerocopy = "0.3"
+async-trait = "0.1.42"
+futures = "0.3.13"
+derivative = "2.2.0"
```

Comment:
I can add another flag for that -- right now these are used based on there being the 'async code' vs async impl. The thought here is you can build a crate foo to depend on fuser with default flags, and implement the actual fs using any impl since the API doesn't change -- the final crate/bin/etc could then add the right async impl flag without a conflict and start it with that.